tests/validation/name-validator-adapter.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { NameValidatorAdapter } from '../../src/application/validation/validators'
```

Comment:
Remove "../../../src/" from import, because the `src` folder is already set as the root of the project in typescript